configuration.go

```diff
@@ -22,7 +22,7 @@ type ConfigurationOption func(*Configuration)
 func NewConfiguration(opts ...ConfigurationOption) *Configuration {
 	cfg := &Configuration{
 		basePath:  "https://api.mux.com",
-		userAgent: "Mux Go | 0.2.2",
+		userAgent: "Mux Go | 0.2.3",
```

Comment:
I'd call this a bump from 0.2.2 to 0.3 because we are not actually returning the model in the `GenericOpenAPIError`

Reply:
Will do, and I'll bump incidents to be 0.4.0